docs/src/sections/SectionUsage.vue

```diff
@@ -130,6 +139,23 @@ import IconSvelte from "../components/IconSvelte.vue"
     </p>
     <CodeExample :examples="reactHook" title="App" />
     <ActualReactApp />
+
+    <h2 id="usage-solid">Solid Primitive</h2>
+    <p>
+      Solid users can use the function <code>createAutoAnimate</code> by
+      importing it from <code>@formkit/auto-animate/solid</code>.
+    </p>
+    <CodeExample :examples="solidPrimitive" title="App" />
+<!--    TODO: publish a version and uncomment it -->
+<!--    <ActualSolidApp />-->
```

Comment:
```suggestion
<ActualSolidApp />
```
When the solidjs code will publish